main.py

```diff
@@ -23,7 +26,22 @@
 
 PRODUCTION = False # added for convenience, all factories eventually created in main and passed to trade_bot
 
+# SERVER SETUP
+def flask_init():
```

Comment:
Where is this going to be called?

Reply:
![Screenshot 2024-02-10 135935](https://github.com/BensGitHub2022/MT5-Expert-Advisor/assets/97774893/01401cde-1389-4daa-9a23-10e83d171537)